langlib/lang.map/src/main/ballerina/map.bal

```diff
@@ -101,20 +105,21 @@ public isolated function forEach(map<Type> m, @isolatedParam function(Type val)
 #
 # + m - the map
 # + func - a predicate to apply to each element to test whether it should be included
-# + return - new map containing members for which `func` evaluates to true
+# + return - new map containing members for which function `func` evaluates to true
```

Comment:
Should such cases also be parameter? 

>An identifier in backticks `X`, when preceded by one of the following words:
> 
> - type
> - service
> - variable
> - var
> - annotation
> - module
> - function
> - parameter
>
> is assumed to be a reference to a Ballerina-defined name of the type indicated by the word.

The name here is of the parameter, right? Not the type?

Reply:
When we have a function type as a parameter, we have used `function` instead of `parameter`. To keep that consistency, I used function here.